src/interpreters/mitie_sklearn_interpreter.py

```diff
@@ -1,13 +1,17 @@
 from mitie import named_entity_extractor, text_categorizer
 from rasa_nlu import Interpreter
 from rasa_nlu.tokenizers.mitie_tokenizer import MITIETokenizer
+import json
+import codecs
 
 
 class MITIESklearnInterpreter(Interpreter):
-    def __init__(self, metadata):
+    def __init__(self, metadata, entity_synonyms=None):
         self.extractor = named_entity_extractor(metadata["entity_extractor"])  # ,metadata["feature_extractor"])
         self.classifier = text_categorizer(metadata["intent_classifier"])  # ,metadata["feature_extractor"])
         self.tokenizer = MITIETokenizer()
+        if entity_synonyms:
+            self.entity_synonyms = json.loads(codecs.open(entity_synonyms, encoding='utf-8').read())
```

Comment:
Since this duplicates the code for every type of backend, how about moving the synonym handling to a separate component and inherit / import the functionality?